config/default/field.field.block_content.uiowa_card.field_uiowa_card_title.yml

```diff
@@ -14,7 +14,7 @@ field_name: field_uiowa_card_title
 entity_type: block_content
 bundle: uiowa_card
 label: Title
-description: ''
+description: 'Choose a headline or title text.'
```

Comment:
Not sure why, but this doesn't seem to be showing up.

Reply:
We needed to render the description in our custom field widget.